### ISSUE TITLE
DNS lookup fails when there is no answer

### ISSUE THREAD

**Issue by westonbelk** [CUCyber/scoreboard#2]
AD/DNS running on Windows Server 2008 R2 (But it probably doesn't matter)
```
DNS: trying 10.0.130.2:53 for A frog.dailsbirdsales.com
Process Process-2:
Traceback (most recent call last):
  File "/usr/lib64/python3.4/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/usr/lib64/python3.4/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.4/site-packages/scoreboard-0.0.1-py3.4.egg/scoreboard/poll.py", line 93, in watch
    if check(opt):
  File "/usr/lib/python3.4/site-packages/scoreboard-0.0.1-py3.4.egg/scoreboard/poll.py", line 33, in check
    return scoreboard.dns.check(**opt)
  File "/usr/lib/python3.4/site-packages/scoreboard-0.0.1-py3.4.egg/scoreboard/dns.py", line 27, in check
    up = response[0].items[0] == answer
AttributeError: 'A' object has no attribute 'items'
```

**Comment by lilyinstarlight** [comment]
Did you pull first? That was fixed in 9dc4135 as far as I know.

**Comment by westonbelk** [comment]
Closed
